aggregator/src/main/java/com/redhat/cloud/notifications/db/AggregationCronjobParameterRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.AggregationCronjobParameters;
+import org.hibernate.Session;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+@ApplicationScoped
+public class AggregationCronjobParameterRepository {
+
+    @Inject
+    Session session;
+
+    @Inject
+    EntityManager entityManager;
+
+    public List<AggregationCronjobParameters> getOrdIdToProceed(LocalDateTime now) {
+        LocalTime currentHour = LocalTime.of(now.getHour(), 0, 0);
+        String query = "SELECT acp FROM AggregationCronjobParameters acp WHERE acp.expectedRunningTime = :currentHour";
+
```

Comment:
What about the old behavior that we need to preserve? If the current hour is midnight UTC, all orgs that did not customize the aggregator execution time should also be included in the list of orgs to aggregate. Currently "midnight UTC" is configured from the `cronjob` scheduling in the YAML file but we could move that scheduling to the app and use a `@ConfigProperty` field to use it here.

Reply:
I just understood that the SQL scripts help preserving the old behavior.